src/testdrive/src/bin/testdrive.rs

```diff
@@ -118,6 +118,13 @@ struct Args {
         value_name = "URL"
     )]
     materialize_url: tokio_postgres::Config,
+    /// materialize SQL internal connection string.
+    #[clap(
```

Comment:
Item No 8 . Strictly speaking there is no need for this and all the other testdrive changes, if you only intend to use this `materialize_url_internal` as a variable inside the .td file . testdrive has a syntax for that purpose already, here is a fragment from another mzcompose file:

``` 
    c.run(
        "testdrive",
        f"--var=ssl-ca={ssl_ca}",
        f"--var=ssl-cert={ssl_cert}",
        f"--var=ssl-key={ssl_key}",
        f"--var=ssl-wrong-cert={ssl_wrong_cert}",
        f"--var=ssl-wrong-key={ssl_wrong_key}",
        *args.filter,
    )
```

Stuff defined as `--var=foo_var=baz` becomes available as `{arg.foo_var}` inside testdrive.

Reply:
I'm somewhat torn on this to be honest. I think that enough tests are going to need to change the system configuration that it's worth it to make the change in the underlying framework rather than for each test that needs to update system configs.